playbooks/roles/deploy-osh/tasks/kubectl-install.yml

```diff
@@ -1,9 +1,11 @@
 ---
 - name: Download latest stable kubectl
+  become: yes
   shell: >-
-    kubectl version || (curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl &&
+    kubectl version --client || (curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl &&
     chmod 0755 ./kubectl &&
-    mv ./kubectl /usr/local/bin/kubectl)
+    mv ./kubectl /usr/local/bin/kubectl &&
```

Comment:
It seems unrelated to this patch, can you separate it into a new PR and add a reason for the extra link?

Reply:
We have this change in ucp branch. I think change is to check explicitly for client version here as version command shows both server and client version. We are updating client aspect so it kind of makes sense though not sure if there is any other reason as change was made by somebody else.

Reply:
I am sorry to be a pain but `this change was made by somebody else` is not a valid `why` reason for me. I can't review without a why. It's also very unrelated to the scope of this patch, so it should be a different patch. If you made this series of different patches with the why, they would have been merged already. It's slowing us down to have `mega patches`.

Reply:
I have provided the reason in my earlier response. I just mentioned that change was not made by me but reason provided earlier kinda makes sense to me.

Now to question of whether its related or not and require a separate PR ..that's a separate discussion (related to issue of having a branch with big change). When we merge that branch, it will bring many additions but expectation is that those should not impact existing CI or ECP mode of dev deployment. 

If you see this will impact existing code in master branch, then I am all ears about the reason and way to address it.

Reply:
The problem is not to provide the reason or not in the review, but in the code/commit message.

But that comment didn't cover the link anyway.

Let's stop nitpicking on this, you have said you will do things after this merge, so I expect this would be part of it?